config/kibana.yml

```diff
@@ -61,6 +61,11 @@
 # must be a positive integer.
 # elasticsearch.requestTimeout: 30000
 
+# List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
+# headers, set this value to nothing/blank (i.e. elasticsearch.requestHeadersWhitelist: )
+# or the empty list (i.e. elasticsearch.requestHeadersWhitelist: [])
```

Comment:
Let's give only one option for this: empty array.  Is there any situation where offering an alternative would be beneficial?